pkg/riff/commands/processor_create.go

```diff
@@ -74,9 +76,12 @@ func NewProcessorCreateCommand(c *cli.Config) *cobra.Command {
 	opts := &ProcessorCreateOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "create",
-		Short:   "<todo>",
-		Example: "<todo>",
+		Use:   "create",
+		Short: "process messages with a function",
```

Comment:
Should mention the processor that is being created. Probably deserves a long description.